content/master/guides/troubleshoot-crossplane.md

```diff
@@ -6,7 +6,7 @@ weight: 306
 
 If you use the Crossplane CLI to install a `Provider` or
 `Configuration` (for example, `crossplane install provider
```

Comment:
ha, i think this has been wrong for a long time, it should be `crossplane xpkg install provider ...`